packages/app/src/runner/screenshot/RemoveClassesDuringScreenshotting.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div :class="classes">
+  <div
+    :class="classes"
```

Comment:
Instead of passing `class` and `style` manually should we just remove `inheritAttrs: true` from here https://github.com/cypress-io/cypress/pull/21254/files#diff-f0a40aeadfe60360756bdb96e57193fb0aed8897794366a826c32c379cff4528R11-R13.

I believe the default is `true`, which passes all the attrs to the root element. docs: https://vuejs.org/api/options-misc.html#inheritattrs

Reply:
Turned out I could revert this change anyway, so I did, `style` here was added while exploring different ways of managing the sizes.